persistence/sql/sqlserver-combining-persistence-wth-transport.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Combining MS SQL Server Persistence and Transport
+component: SqlPersistence
+reviewed: 2024-06-10
+related:
+  - persistence/sql/dialect-mysql
+  - transports/sql
+---
+
+## Connection behavior
+
+When combining SQL Server transport and persistence using the Sql dialect, the connection behaves differently based on whether the [Outbox](/nservicebus/outbox/) is used or not. This influences where the saga data is stored.
+
+### Without Outbox
+
+SQL Transport<br/>TransactionMode | SQL Persistence<br/>with Sql dialect | Connection sharing | Saga location
+:-:|:-:|:-:|:-:
+TransactionScope |  ✅| SQLT transaction is promoted to distributed transaction | Persistence DB <sup>1</sup>
```

Comment:
What is `SQLT`, according to google https://www.google.com/search?q=sqlt
I guess my point is that the acronym, is known for something else, I think we need to change it

Reply:
I have changed it to SQL Transport